src/auth.js

```diff
@@ -130,17 +131,16 @@ export default {
 
   async _getSignerTypes (accountId) {
     const result = {}
-    let response = {}
-
+    let signer = {}
     try {
-      response = await accounts.getSignerById(accountId)
+      signer = (await Sdk.horizon.account.getSigner(accountId)).data
```

Comment:
move the whole stuff to `try` block and use destructuring here, I mean, instead of 

```
signer = (await Sdk.horizon.account.getSigner(accountId)).data
```

do

```
const { data: signer } = await ...
```